ogr/ogrsf_frmts/gml/embedded_resources_gen2.c

```diff
@@ -0,0 +1,36 @@
+// File generated by generate_embedded_resources.py. DO NOT EDIT
```

Comment:
My experience with keeping generated sources under version control hasn't been great, so consider the alternative of having CMake generate these files somewhere inside `${CMAKE_CURRENT_BINARY_DIR}`. (Not suggesting to replace the Python script with CMake-code, just that CMake sets up a task that calls the Python script to generate the files.)

Reply:
I'll let that as an exercise for later. We do not add new .gfs files frequently. Last time was years ago

Reply:
Actually, we don't have a Python build requirement currently